packages/mux-player/tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "esModuleInterop": true,
     "skipLibCheck": false,
     "forceConsistentCasingInFileNames": true,
-    "typeRoots": ["node_modules/@types", "../../types/media-chrome.d.ts"]
```

Comment:
question(blocking): Why were these types things changed for this PR? This might be right (I'd want to make sure for all the different use cases we have), but unless this is needed for the PR, these changes should be backed out and added to a separate PR.

Reply:
this was needed to make it build, because the MediaErrorDialog class is imported.

the media-chrome.d.ts file was excluding the types that are now available in the Media Chrome pkg itself

```ts
declare module 'media-chrome' {
  export class MediaTheme extends HTMLElement {}
  export class MediaController extends HTMLElement {
    media: HTMLVideoElement;
  }
}
```